source/03-components/accordion-item/_accordion-item.scss

```diff
@@ -0,0 +1,73 @@
+// Component: accordion-item
+
+@use '00-config' as *;
+
+$accordion-item-border-color: gesso-grayscale(gray-2) !default;
+$accordion-item-icon-color: gesso-color(ui, generic, accent) !default;
+$accordion-item-icon-size: 20px !default;
+
+.accordion-item {
+  border-bottom: solid 1px $accordion-item-border-color;
+}
+
+.accordion-item__heading {
+  font-weight: gesso-font-weight(bold);
+  margin: 0;
+}
+
+.accordion-item__toggle {
+  background-color: transparent;
+  border: 0;
+  cursor: pointer;
+  display: flex;
+  justify-content: space-between;
+  padding: rem(gesso-spacing(2)) rem(gesso-spacing(8)) rem(gesso-spacing(2))
+    rem(gesso-spacing(2));
+  position: relative;
+  text-align: left;
+  transition: all gesso-duration(short) gesso-easing(ease-out);
+  width: 100%;
+}
+
+.accordion-item__drawer {
+  background: gesso-grayscale(white);
+  display: none;
+}
+
+.accordion-item__drawer-inner {
+  padding: 0 rem(gesso-spacing(8)) rem(gesso-spacing(2)) rem(gesso-spacing(2));
+}
```

Comment:
Should we add the following to prevent content from adding too much margin at the bottom??

```scss
.accordion-item__drawer-inner {
  > :last-child {
    margin-bottom: 0;
  }
}
```